examples/js/loaders/GLTFLoader.js

```diff
@@ -147,6 +146,12 @@ THREE.GLTFLoader = ( function () {
 
 				}
 
+				if ( json.extensionsUsed.indexOf( EXTENSIONS.MSFT_TEXTURE_DDS ) >= 0 ) {
+
+					extensions[ EXTENSIONS.MSFT_TEXTURE_DDS ] = "MSFT_texture_dds";
```

Comment:
How about making GLTFTextureDDSExtension() for the consistency with other extensions? And if you instantiate DDSLoader there, it can be reused.

    function GLTFTextureDDSExtension() {

        this.name = EXTENSIONS.MSFT_TEXTURE_DDS;
        this.loader = new THREE.DDSLoader();

    }